app/models/manageiq/providers/vmware/cloud_manager.rb

```diff
@@ -84,4 +84,11 @@ def vm_restart(vm, _options = {})
   def self.display_name(number = 1)
     n_('Cloud Provider (VMware vCloud)', 'Cloud Providers (VMware vCloud)', number)
   end
+
+  def vm_create_snapshot(vm, options = {})
+    options[:quiesce] = 'false'
```

Comment:
I would be safer if you do it like it's done on the infra manager: https://github.com/ManageIQ/manageiq-providers-vmware/blob/master/app/models/manageiq/providers/vmware/infra_manager.rb#L334-L340

So basically you set all default options and then only override those that were specifically passed. Also, I'm not sure why would you pass `'false'` (string) and not `false` (boolean), is there a reason?

Reply:
I agree with @miha-plesko. Set defaults for memory and quiesce (both false) and then merge with given options to allow users to override them.